subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3444,6 +3490,49 @@ static int cmd_create_broadcast_sink(const struct shell *sh, size_t argc, char *
 	return 0;
 }
 
+static int cmd_create_sink_by_name(const struct shell *sh, size_t argc, char *argv[])
+{
+	char *broadcast_name;
+	int err = 0;
+
+	broadcast_name = argv[1];
+	if (!IN_RANGE(strlen(broadcast_name), BT_AUDIO_BROADCAST_NAME_LEN_MIN,
+		      BT_AUDIO_BROADCAST_NAME_LEN_MAX)) {
+
```

Comment:
```suggestion
		      BT_AUDIO_BROADCAST_NAME_LEN_MAX)) {
```

Reply:
I can’t see what change you suggested. Could you please clarify it?

Reply:
Remove the empty line :)

Reply:
Ok :)